airflow/operators/sql.py

```diff
@@ -467,6 +467,218 @@ def push(self, meta_data):
         self.log.info("Log from %s:\n%s", self.dag_id, info)
 
 
+def _get_failed_tests(checks):
+    failed_tests = []
+    for check, check_values in checks.items():
+        if not check_values["success"]:
+            failed_tests.append(
+                f"\tCheck: {check}, "
+                f"Pass Value: {check_values['pass_value']}, "
+                f"Result: {check_values['result']}\n"
+            )
+    return failed_tests
```

Comment:
```suggestion
def _get_failed_tests(checks):
    return [
        f"\tCheck: {check}, "
        f"Pass Value: {check_values['pass_value']}, "
        f"Result: {check_values['result']}\n"
        for check, check_values in checks.items()
        if not check_values["success"]
    ]
```

Reply:
But this string does not seem to be used at all. Maybe it should simply be like this?

```python
def _all_success(checks):
    return all(values["success"] for values in checks.values())
```